### ISSUE TITLE
`ts-decorate` and `ts-metadata` helpers not found in Script source type

### ISSUE THREAD

**Issue by JSerFeng** [swc-project/swc#7118]
### Describe the bug

source type: module

```ts
...
var _tsDecorate = require("@swc/helpers/lib/_ts_decorate.js").default;
```
source type: script

```ts
...
var _ts_decorate = require("@swc/helpers").ts_decorate;
```

require("@swc/helpers").ts_decorate is `undefined`

### Input code

```typescript
@aaa
class a {

}
```

### Config

```json
{
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "tsx": true,
      "decorators": true
    },
    "externalHelpers": true,
    "target": "es2022"
  },
  "module": {
    "type": "commonjs"
  },
  "minify": false,
  "isModule": false
}
```

### Playground link

https://play.swc.rs/?version=1.3.41&code=H4sIAAAAAAAAA3NITEzkSs5JLC5WSFSo5uKqBQAgLzirEQAAAA%3D%3D&config=H4sIAAAAAAAAA02OMQ6DMAxFd06BPHeoMvYEXXoIK5gqKCSRbSQixN2b0EC7%2Bf%2F3v%2B2t63uYxMKj38pYREIW4ksXR3JQXIsDmhOJZZcUbidVqUh5ocsayEZGjSyNHGD%2FcqBViQP6J%2FlEV6RBRX6T1lsk5m4MdK0IcxwWT78%2F6y81Z%2BM8xzDJX9IFN%2BbCRvRyLAYnr7N%2BmN3%2BAfGuH6b5AAAA

### Expected behavior

ts_decorate is defined

### Actual behavior

ts_decorate is undefined

### Version

1.3.41

**Comment by JSerFeng** [comment]
You can leave this to me 🥰

**Comment by kdy1** [comment]
Thank you!

**Comment by magic-akari** [comment]
This will be fixed in #7147

**Comment by JSerFeng** [comment]
Thanks !
